lib/p2p/packets/packetUtils.ts

```diff
@@ -1,20 +1,17 @@
 import PacketType from './PacketType';
 import Packet from './Packet';
-import OrderPacket from './OrderPacket';
-import HelloPacket from './HelloPacket';
-import PingPacket from './PingPacket';
-import PongPacket from './PongPacket';
+import * as packetTypes from './types';
 
-const fromRaw = (type:string, body:string): Packet|void => {
+const fromRaw = (type:string, packet:string): Packet|void => {
```

Comment:
There should be spaces between the colon and the type, like `type: string`, for consistency.

Reply:
Fixed.